rmgpy/solver/base.pyx

```diff
@@ -700,7 +706,10 @@ cdef class ReactionSystem(DASx):
         unimolecularThreshold = self.unimolecularThreshold
         bimolecularThreshold = self.bimolecularThreshold
         trimolecularThreshold = self.trimolecularThreshold
-        
+
+        self.MW = numpy.array([spc.molecularWeight.value_si for spc in coreSpecies+edgeSpecies])
```

Comment:
Iterating over two lists by using `+` is not great because it creates a new list temporarily. Unfortunately I don't know what the optimal way is in cython. Since this happens once per simulation (?), we can probably leave it.

Reply:
I've switched to itertools.chain